packages/jupyter-chat/src/components/chat.tsx

```diff
@@ -134,6 +121,10 @@ export namespace Chat {
      * Attachment opener registry.
      */
     attachmentOpenerRegistry?: IAttachmentOpenerRegistry;
+    /**
+     * The input toolbar registry
+     */
+    inputToolbarRegistry?: IInputToolbarRegistry;
```

Comment:
Reference to https://github.com/jupyterlab/jupyter-chat/issues/172, this property could also be added to a `ReactContext`.